rllib/agents/a3c/tests/test_a3c.py

```diff
@@ -25,13 +25,15 @@ def test_a3c_compilation(self):
 
         # Test against all frameworks.
         for _ in framework_iterator(config):
-            for env in ["CartPole-v0", "Pendulum-v0", "PongDeterministic-v0"]:
+            for env in ["CartPole-v1", "Pendulum-v0", "PongDeterministic-v0"]:
```

Comment:
V1 Cartpole goes to 500 reward, not 200, but change the results fyi

Reply:
That's ok, it's just a "compilation" test. No learning progress checked here.